pkg/diff/cache/interface.go

```diff
@@ -55,7 +55,7 @@ type CommonOptions struct {
 }
 
 func (options *CommonOptions) Setup(fs *pflag.FlagSet) {
-	fs.DurationVar(&options.PatchTtl, "diff-cache-patch-ttl", time.Minute*10, "duration for which patch cache remains (0 to disable TTL)")
+	fs.DurationVar(&options.PatchTtl, "diff-cache-patch-ttl", 0, "duration for which patch cache remains (0 to disable TTL)")
```

Comment:
For future reference: We set this to 0 by default because each patch is only fetched once, and the only chance this can get reused is when consumer and controller are deployed as the same process.

Reply:
有两种方法修：

1. memory_wrapper 就直接关闭patchCache，只支持snapshotCache
2. 额外再新增一个参数用于控制patch memory cache，比如 diff-cache-patch-wrapper-enable，但是原来那个diff-cache-wrapper-enable就只控制snapshot有点奇怪，如果diff-cache-wrapper-enable也改名的话就比较麻烦了